internal/kafka/internal/services/kafka.go

```diff
@@ -193,30 +194,30 @@ func (k *kafkaService) AssignInstanceType(kafkaRequest *dbapi.KafkaRequest) (typ
 		return types.STANDARD, nil
 	}
 
-	return types.EVAL, nil
+	return types.DEVELOPER, nil
 }
 
 // reserveQuota - reserves quota for the given kafka request. If a RHOSAK quota has been assigned, it will try to reserve RHOSAK quota, otherwise it will try with RHOSAKTrial
 func (k *kafkaService) reserveQuota(kafkaRequest *dbapi.KafkaRequest) (subscriptionId string, err *errors.ServiceError) {
-	if kafkaRequest.InstanceType == types.EVAL.String() {
-		if !k.kafkaConfig.Quota.AllowEvaluatorInstance {
-			return "", errors.NewWithCause(errors.ErrorForbidden, err, "kafka eval instances are not allowed")
+	if kafkaRequest.InstanceType == types.DEVELOPER.String() {
+		if !k.kafkaConfig.Quota.AllowDeveloperInstance {
+			return "", errors.NewWithCause(errors.ErrorForbidden, err, "kafka developer instances are not allowed")
 		}
 
-		// Only one EVAL instance is admitted. Let's check if the user already owns one
+		// Only one DEVELOPER instance is admitted. Let's check if the user already owns one
 		dbConn := k.connectionFactory.New()
 		var count int64
 		if err := dbConn.Model(&dbapi.KafkaRequest{}).
-			Where("instance_type = ?", types.EVAL).
+			Where("instance_type = ?", types.DEVELOPER).
 			Where("owner = ?", kafkaRequest.Owner).
 			Where("organisation_id = ?", kafkaRequest.OrganisationId).
 			Count(&count).
 			Error; err != nil {
-			return "", errors.NewWithCause(errors.ErrorGeneral, err, "failed to count kafka eval instances")
+			return "", errors.NewWithCause(errors.ErrorGeneral, err, "failed to count kafka developer instances")
```

Comment:
If these errors are returned to the caller (maybe in the UI), I believe we should use the `display name` instead of `developer` here.

Reply:
done